src/app.js

```diff
@@ -4,6 +4,14 @@ const http = require('http')
   , server = http.createServer(app)
   , bodyParser = require('body-parser');
 
+// Signal handling
+process.on('SIGTERM', () => {
+  console.log('Received SIGTERM signal. Gracefully stopping web server.');
```

Comment:
Is it reasonable to use `console.log` in there? If not, what should I use instead? Maybe just dropping the log altogether?

Reply:
Isn't a big deal but, it's probably better to create an external logger to manage this king of things. Otherwise, we can only log in DEBUG 🤷‍♂️ 

I could be fine to watch this log in case of crash loop